svcplugin/consulextension/consul.go

```diff
@@ -19,11 +18,13 @@ func init() {
 	bridge.Register(new(Factory), "consul")
 }
 
+/* TODO: Fix this for check_script
```

Comment:
I suggest that we remove this commented out code. It doesn't look good in an open sourced repo. If we need to fix a script to make it work then may be also fix the script as part of this PR?